opus/application/static_media/js/browse.js

```diff
@@ -312,29 +277,102 @@ var o_browse = {
             // we will hide page status loader from infiniteScroll if op-page-loading-status loader is spinning
             $(".op-page-loading-status > .loader").show();
             let orderBy =  $(this).data("slug");
+            let targetSlug = orderBy;
 
+            // get order of opusid when table header is clicked
+            let hash = o_hash.getHashArray();
+            let opusidOrder = (hash.order || hash.order.match(/(-?opusid)/)) ? hash.order.match(/(-?opusid)/)[0] : "opusid";
```

Comment:
I don't think you want to use || here. What this is currently saying is "If hash.order is not empty...retrieve hash.order.match...[0]". I think what you're trying to say is "If hash.order is not empty AND hash.order.match... is true THEN return hash.order.match...[0]". && would work better.